internal/codespaces/grpc/client.go

```diff
@@ -68,7 +70,8 @@ func Connect(ctx context.Context, session liveshareSession, token string) (*Clie
 		grpc.WithTransportCredentials(insecure.NewCredentials()),
 		grpc.WithBlock(),
 	}
-	ctx, _ = context.WithTimeout(ctx, serverConnectionTimeout)
+	ctx, cancel = context.WithTimeout(ctx, serverConnectionTimeout)
+	defer cancel()
```

Comment:
I think this will cause some issues with the long-lasting client activity gRPC call, but we can handle that when it comes time for that PR in the future. Discarding the cancel was throwing a linting error so I went ahead and used it here.